src/Microsoft.AspNetCore.DataProtection.Extensions/DataProtectionProvider.cs

```diff
@@ -8,21 +8,19 @@
 namespace Microsoft.AspNetCore.DataProtection
 {
     /// <summary>
-    /// A simple implementation of an <see cref="IDataProtectionProvider"/> where keys are stored
-    /// at a particular location on the file system.
+    /// An entry point class for creating an <see cref="IDataProtectionProvider"/> where keys are stored
+    /// at a particular location on the file system on non dependency injection enabled scenarios.
```

Comment:
I'd put the `on non dependency injection enabled scenarios` part in a `<remarks>` section. Also, the phrasing is a bit nerdy. Perhaps something like `<remarks>Use this method when not using dependency injection to provide the service to the application.</remarks>`?